Cargo.toml

```diff
@@ -58,7 +58,7 @@ futures = "0.3.30"
 # TODO generic-array should not be a direct dependency
 # https://github.com/EspressoSystems/HotShot/issues/1850
 generic-array = { version = "0.14.7", features = ["serde"] }
-hotshot-types = { git = "https://github.com/EspressoSystems/hotshot-types" }
+hotshot-types = { git = "https://github.com/EspressoSystems/hotshot-types", branch = "jparr721/issue-2732" }
```

Comment:
This needs a fast-follow when this merges: https://github.com/EspressoSystems/hotshot-types/pull/1. Tracking: https://github.com/EspressoSystems/HotShot/issues/2759